system-test/read-rows-acceptance-tests.ts

```diff
@@ -87,6 +87,9 @@ describe('Read Row Acceptance tests', function() {
           objectMode: true,
         });
 
+        /* tslint:disable-next-line */
+        (stream as any).abort = function() {};
```

Comment:
I believe nodejs-common has an `AbortableDuplex` interface that should work here.

Reply:
@JustinBeckwith could you take a look if I went about this the right way? I had to add a devDependency on common, and cast the PassThrough streams to `Duplex` streams. It started to feel wrong :)